chapter5.Rmd

```diff
@@ -88,6 +88,7 @@ As mentioned before, lists can also contain other lists. This works just the sam
 *** =instructions
 - Construct a list, named `my_super_list`, that now contains the four predefined variables listed in the sample code (in the same order).
 - Print the structure of `my_super_list` with the [`str()`](http://www.rdocumentation.org/packages/utils/functions/str) function.
+-Enter the variables in the following order: `my_vector`, `my_matrix`, `my_factor`, `my_list`
```

Comment:
Yea, this should probably be made clear, but a few points worth mentioning here.

1. You may need a space after the dash so that this renders correctly as a list item in markdown
2. This question does a bad job at it, but typically we now try to map exactly one instruction to one comment (i.e. placeholder) in the R script provided. Can you improve this?